stock-tracker/src/loading.css

```diff
@@ -0,0 +1,33 @@
+.lds-ripple {
+  display: flex;
+  margin: 10% auto;
+  position: relative;
+  width: 64px;
+  height: 64px;
+}
+.lds-ripple div {
+  position: absolute;
+  border: 4px solid #fff;
+  opacity: 1;
+  border-radius: 50%;
+  animation: lds-ripple 1s cubic-bezier(0, 0.2, 0.8, 1) infinite;
+}
+.lds-ripple div:nth-child(2) {
```

Comment:
Seeing as you now have a specific style for one of the 2 divs, why not just apply classnames to them?
`lds-ripple__one` and `lds-ripple__two` or something of the sort?